frontend/src/metabase/query_builder/containers/QueryBuilder.unsaved-changes-warning.unit.spec.tsx

```diff
@@ -39,6 +39,14 @@ import {
 registerVisualizations();
 
 describe("QueryBuilder - unsaved changes warning", () => {
+  beforeEach(() => {
+    HTMLElement.prototype.scrollBy = jest.fn();
```

Comment:
should we clear it in afterEach?

Reply:
I did it in a subsequent PR to not have CI running one more time here: https://github.com/metabase/metabase/pull/42821/commits/be72092f6b3e9e92c53c6ce831477d7e3e06958c
(it goes to a feature branch anyway)